### PR TITLE
 Allow user to specify default address pools for docker networks

### PR DESCRIPTION
**Pull request by selansen**
This is separate commit for CLI files to address PR https://github.com/moby/moby/pull/36054

Signed-off-by: selansen <elango.siva@docker.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: docs/reference/commandline/dockerd.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ Options:
   -D, --debug                                 Enable debug mode
       --default-gateway ip                    Container default gateway IPv4 address
       --default-gateway-v6 ip                 Container default gateway IPv6 address
+      --default-address-pools                 Set the default address pools for local node networks
```
Comment (thaJeztah) [inline docs/reference/commandline/dockerd.md]: This should be singular now

Also, I think this page has an example `daemon.json`; can you update that one as well (with some example values)

File: man/dockerd.8.md [hunk 2]
```diff
@@ -22,6 +22,7 @@ dockerd - Enable daemon mode
 [**-D**|**--debug**]
 [**--default-gateway**[=*DEFAULT-GATEWAY*]]
 [**--default-gateway-v6**[=*DEFAULT-GATEWAY-V6*]]
+[**--default-address-pools**[=*DEFAULT-ADDRESS-POOLS*]]
```
Comment (thaJeztah) [inline man/dockerd.8.md]: same here

**Comment by selansen** [comment]
will correct it

File: docs/reference/commandline/dockerd.md [hunk 3]
```diff
@@ -1349,6 +1350,8 @@ This is a full example of the allowed configuration options on Linux:
 			]
 		}
 	}
+	"default-address-pools":[{"base":"172.80.0.0/16","size":24},
```
Comment (thaJeztah) [inline docs/reference/commandline/dockerd.md]: Looks like there's a comma missing in the JSON (it shows as invalid)

**Comment by selansen** [comment]
corrected it.

**Pull request merged**